manifests/w/WowUp/Wowup/2.4.6/WowUp.Wowup.installer.yaml

```diff
@@ -0,0 +1,18 @@
+# Created with YamlCreate.ps1 v2.0.0 $debug=QUSU.7-1-4
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+
+PackageIdentifier: WowUp.Wowup
+PackageVersion: 2.4.6
+InstallerLocale: en-US
+MinimumOSVersion: 10.0.0.0
+InstallerType: nullsoft
+Scope: user
+InstallModes:
+- silent
+UpgradeBehavior: install
+Installers:
+- Architecture: x64
+  InstallerUrl: https://github.com/WowUp/WowUp/releases/download/v2.4.6/WowUp-2.4.6.exe
+  InstallerSha256: BB10A2A4B02CAD5695F80E94F051D733426EC483CBFD876CDDA5E596F100EE58
```

Comment:
```suggestion
  InstallerUrl: https://github.com/WowUp/WowUp/releases/download/v2.4.6/WowUp-Setup-2.4.6.exe
  InstallerSha256: 36B2AB9666A31863C2FA37C63C4F835587DA3882540BF06BDD358E316439CDDB
```

Reply:
Make sure to use the URL that has `WowUp-Setup-` in the file name because the one you used is the portable installer.